src/lib/proxyCache.js

```diff
@@ -41,6 +43,7 @@ const getFSPProxy = async (dfspId) => {
 }
 
 const checkSameCreditorDebtorProxy = async (debtorDfspId, creditorDfspId) => {
+  logger.debug(`Checking if ${debtorDfspId} and ${creditorDfspId} are using the same proxy`)
```

Comment:
I'd definitely recommend avoid using template strings as log message. The main reason is debugging and searching in logs. Let me try to explain my point:
1. When I found some logs (say, using Grafana), and I want to find appropriate place in code:
![image](https://github.com/user-attachments/assets/6c57f774-b59a-465b-aded-e5c08eff0be3)
It's much easier and faster to do it if that log message is "static" string, and I don't need to remember that, say, 3rd word is "dynamic", and I need to search only by first 2 words.  In this particular log, in Grafana I'll have something like:
`Checking if 1234-3442345 and 90905-23432-..... are using the same proxy`, right?  So I need to search only by `Checking if `. And  I can get different matches (other places in code). It's not a huge issue, of course, but anyway.

So from my perspective, having log like:
```
logger.debug('Checking if debtor and creditor are using the same proxy', {  debtorDfspId, creditorDfspId })
```
will simplify such searching

2. Vice versa approach: when you want to filter logs to find exactly this line (using Grafana or grep). Again, I can search only by `Checking if`, which not exactly what I need.

3. If you use template string, you have to use `.isDebudEnabled` approach:
 ```
 logger.isDebugEnabled && logger.debug(`Checking if ${debtorDfspId} and ${creditorDfspId} are using the same proxy`)
 ```
 Personally, I don't think it affects performance much, but I had several "recommendation" about that from Vijay.
 
 As  a summary, I don't see any reason to use template string as log message, if we have possibility to pass second argument as an object with detailed information.

Reply:
done